src/App.tsx

```diff
@@ -13,8 +13,10 @@ export class App extends React.Component<{}, State> {
     movies: moviesFromServer,
   };
 
-  addMovie = (/* movie: Movie */) => {
-    // put your code here
+  addMovie = (movie: Movie) => {
+    this.setState((prev) => ({
+      movies: [movie, ...prev.movies],
```

Comment:
better to add to the end of the list

```suggestion
      movies: [ ...prev.movies, movie],
```